src/home/marketing/paragraph-one.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+
+export const ParagraphOne = () => <Typography paragraph>
```

Comment:
Okay... so __this__ is the actual content of each section and you use the same component for both.

Let's not do that. Why? Because we'll probably have different text in each section and we'll have to split it up then anyways. So let's create separate components for each section.

Reply:
Done!